README.md

```diff
@@ -39,6 +39,11 @@ colcon build
 ```
 
 ## Run
+The `TURTLEBOT3_MODEL` environment variable is optional when running both robot and simulation application. Default value is `waffle_pi`. Valid values are `burger`, `waffle`, and `waffle_pi`. Set it by
```

Comment:
For simulation application, TURTLEBOT3_MODEL is set by ```turtlebot3_description_reduced_mesh``` launch file [here](https://github.com/aws-robotics/aws-robomaker-sample-application-helloworld/blob/21a37f921e77cd0bf1dae49142dc0f4cf050e40b/simulation_ws/src/hello_world_simulation/launch/empty_world.launch#L18) correct?

Reply:
I think the environment variable isn't set anywhere, the launch file you mentioned also have this optional TURTLEBOT3_MODEL with default `waffle_pi` (see [here](https://github.com/aws-robotics/turtlebot3-description-reduced-mesh/blob/ros1/launch/spawn_turtlebot.launch#L3))

Reply:
I think so, but, regardless, the environment variable should be set here at the top-level.